phpcs.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<ruleset name="PHP_CodeSniffer">
+    <description>PHP_CodeSniffer configuration</description>
+    <rule ref="PSR12">
+        <exclude name="Squiz.Classes.ValidClassName" />
+        <exclude name="Generic.Files.LineLength" />
+        <exclude name="Squiz.Commenting.FunctionComment.IncorrectParamVarName" />
+        <exclude name="Squiz.Commenting.FunctionComment.InvalidReturn" />
+    </rule>
+
+    <arg name="bootstrap" value="vendor/zebra-north/phpcs-short-types/short-types.php"/>
+
+    <rule ref="Modernize" />
+    <rule ref="NormalizedArrays" />
+
+    <!-- exclude our migrations directory from the violation check-->
+    <exclude-pattern>*.blade.php</exclude-pattern>
+
+    <!-- ignore warnings and display ERRORS only -->
+    <arg  value="np"/>
+
+    <rule ref="Universal.Namespaces.DisallowCurlyBraceSyntax">
+        <exclude name="Universal.Namespaces.DisallowCurlyBraceSyntax.Forbidden" />
+    </rule>
+
+    <rule ref="Squiz.Commenting.FunctionComment">
+        <exclude name="Squiz.Commenting.FunctionComment.MissingParamTag" />
+        <exclude name="Squiz.Commenting.FunctionComment.MissingParamComment" />
+        <exclude name="Squiz.Commenting.FunctionComment.ParamCommentFullStop" />
+        <exclude name="Squiz.Commenting.FunctionComment.EmptyThrows" />
+        <exclude name="Squiz.Commenting.VariableComment.MissingReturn" />
+        <exclude name="Squiz.Commenting.FunctionComment.ParamCommentNotCapital" />
+    </rule>
+
+    <rule ref="Squiz.Commenting.FunctionCommentThrowTag">
+        <exclude name="Squiz.Commenting.FunctionCommentThrowTag.Missing" />
+    </rule>
+
+    <rule ref="Squiz.Commenting.ClassComment" />
+
+    <rule ref="Squiz.Commenting.VariableComment" >
+        <!-- Put back in at some point -->
+        <exclude name="Squiz.Commenting.VariableComment.Missing" />
+    </rule>
+
+
```

Comment:
Why all the empty lines?

Reply:
It was taken directly from https://github.com/ITK-Leantime/leantime/blob/master/phpcs.xml. I have removed empty lines.